code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -834,4 +850,4 @@
 	access = get_all_accesses()
 	access += get_all_centcom_access()
 
-/obj/item/weapon/card/id/death_commando_
\ No newline at end of file
+/obj/item/weapon/card/id/death_commando_
```

Comment:
I don't think this is supposed to have a _ at the end
it also seems to not have any variables defined?

Reply:
it's a mystery
will purge